### ISSUE TITLE
Logits decode time is increasing over time using the streaming feature

### ISSUE THREAD

**Issue by carlfm01** [mozilla/DeepSpeech#1757]
Hi, I finally compiled it for Windows and currently using the native client from C#, the problem is with the `DS_IntermediateDecode` of `deepspeech.cc`, I'm feeding the streaming feature from the windows sound output and calling `DS_IntermediateDecode` for example every 2 seconds, in the first seconds of streaming the decode is taking less than a second (acceptable), but if I feed 20 seconds the decoding time noticeable increases and takes more than a few seconds, if I keep feeding seconds the decode time keeps increasing too.
I tried to split the logits to just evaluate the current context logits and the past context but I failed, unfortunately I'm new to C++. I'm doing wrong trying to split the logits? This is a possible issue or that is the expected behavior?

Here is where I tried to split the logits: https://github.com/mozilla/DeepSpeech/blob/de279168ecfa47e79d4b155c4c529267441a051a/native_client/deepspeech.cc#L332

Thanks.

- **Have I written custom code (as opposed to running examples on an unmodified clone of the repository)**: No, just few compiler conditionals to make it compiles on Windows
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows 10
- **TensorFlow installed from (our builds, or upstream TensorFlow)**: --
- **TensorFlow version (use command below)**:It prints unknown, maybe because I'm on Windows, however I compiled the current master of "mozilla/tensorflow"
- **Python version**: NA
- **Bazel version (if compiling from source)**:NA
- **GCC/Compiler version (if compiling from source)**: MSVC
- **CUDA/cuDNN version**: NA
- **GPU model and memory**: NA
- **Exact command to reproduce**: NA

**Comment by reuben** [comment]
This is the expected behavior as we don't have a streaming decoder yet.
